index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/5116129a4d.js" crossorigin="anonymous"></script>
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+   <section class="header">
+    <div class="name">Daniel</div>
+    <div class="hamburger"><i class="fas fa-hamburger"></i></div>
+   </section>
+
+   <section class="headline">
+    <h1 class="intro">Hey There. I'm Daniel</h1>
+   <p class="developer">I'm a Software developer</p>
```

Comment:
- [ ] Since this is in essence a sub-heading, using a `<h2>` or `<h3>` tag would be a more meaningful description of the text `I'm a software developer`. Kindly alter this code to use a hx tag.